ocs_ci/helpers/helpers.py

```diff
@@ -4103,6 +4110,29 @@ def create_reclaim_space_cronjob(
     return ocs_obj
 
 
+def create_csi_addons_global_timeout_configmap():
+    """
+    Create Global timeout config for reclaimspace job
+
+    Returns:
+        bool: Returns timeout value and cm object if the operation was successful, False otherwise.
+    """
+    logging.info("Creating config map for reclaim space global timeout")
+    global_timeout_configmap = templating.load_yaml(
+        constants.CSI_RBD_RECLAIM_SPACE_CONFIGMAP_YAML
+    )
+
+    # timeout in str format
+    logging.info(f"global_timeout_configmap is {global_timeout_configmap}")
+    timeout = global_timeout_configmap["data"]["reclaim-space-timeout"]
+    logging.info(f"timeout {timeout}")
+    cm_obj = create_resource(**global_timeout_configmap)
+    if cm_obj:
+        return timeout, cm_obj
+    else:
+        return False
```

Comment:
Should return two values to avoid unpacking error during function call eg. False, None